mavros/launch/px4_config.yaml

```diff
@@ -138,144 +152,161 @@ mission:
-    field_of_view: 0.0  # XXX TODO
-    send_tf: true
-    sensor_position: {x:  0.0, y:  0.0, z:  -0.1}
-  lidarlite_pub:
-    id: 1
-    frame_id: "lidarlite_laser"
-    orientation: PITCH_270
-    field_of_view: 0.0  # XXX TODO
-    send_tf: true
-    sensor_position: {x:  0.0, y:  0.0, z:  -0.1}
-  sonar_1_sub:
-    subscriber: true
-    id: 2
-    orientation: PITCH_270
-    horizontal_fov_ratio: 1.0   # horizontal_fov = horizontal_fov_ratio * msg.field_of_view
-    vertical_fov_ratio: 1.0     # vertical_fov = vertical_fov_ratio * msg.field_of_view
-    # custom_orientation:       # Used for orientation == CUSTOM
-    #   roll: 0
-    #   pitch: 270
-    #   yaw: 0
-  laser_1_sub:
-    subscriber: true
-    id: 3
-    orientation: PITCH_270
+/mavros/distance_sensor:
+  ros__parameters:
+    config: |
+      hrlv_ez4_pub:
+        id: 0
+        frame_id: "hrlv_ez4_sonar"
+        orientation: PITCH_270 # RPY:{0.0, 270.0, 0.0} - downward-facing
+        field_of_view: 0.0  # XXX TODO
+        send_tf: true
+        sensor_position: {x:  0.0, y:  0.0, z:  -0.1}
+      lidarlite_pub:
+        id: 1
+        frame_id: "lidarlite_laser"
+        orientation: PITCH_270
+        field_of_view: 0.0  # XXX TODO
+        send_tf: true
+        sensor_position: {x:  0.0, y:  0.0, z:  -0.1}
+      sonar_1_sub:
+        subscriber: true
+        id: 2
+        orientation: PITCH_270
+        horizontal_fov_ratio: 1.0   # horizontal_fov = horizontal_fov_ratio * msg.field_of_view
+        vertical_fov_ratio: 1.0     # vertical_fov = vertical_fov_ratio * msg.field_of_view
+        # custom_orientation:       # Used for orientation == CUSTOM
+        #   roll: 0
+        #   pitch: 270
+        #   yaw: 0
+      laser_1_sub:
+        subscriber: true
+        id: 3
+        orientation: PITCH_270
 
 # image_pub
-image:
-  frame_id: "px4flow"
+/mavros/image:
+  ros__parameters:
+    frame_id: "px4flow"
+
 
 # fake_gps
-fake_gps:
-  # select data source
-  use_mocap: true         # ~mocap/pose
-  mocap_transform: true   # ~mocap/tf instead of pose
-  use_vision: false       # ~vision (pose)
-  # origin (default: Zürich)
-  geo_origin:
-    lat: 47.3667          # latitude [degrees]
-    lon: 8.5500           # longitude [degrees]
-    alt: 408.0            # altitude (height over the WGS-84 ellipsoid) [meters]
-  eph: 2.0
-  epv: 2.0
-  satellites_visible: 5   # virtual number of visible satellites
-  fix_type: 3             # type of GPS fix (default: 3D)
-  tf:
-    listen: false
-    send: false           # send TF?
-    frame_id: "map"       # TF frame_id
-    child_frame_id: "fix" # TF child_frame_id
-    rate_limit: 10.0      # TF rate
-  gps_rate: 5.0           # GPS data publishing rate
+/mavros/fake_gps:
+  ros__parameters:
+    # select data source
+    use_mocap: true         # ~mocap/pose
+    mocap_transform: true   # ~mocap/tf instead of pose
+    use_vision: false       # ~vision (pose)
+    # origin (default: Zürich)
+    geo_origin:
```

Comment:
https://github.com/mavlink/mavros/blob/6e86500318385b5770653c6e47e3c0cdda2e3181/mavros_extras/src/plugins/fake_gps.cpp#L133-L144

So it should look like:
```
fake_gps:
  ros__parameters:
    geo_origin.lat: 47.3667
```

Reply:
OK, I see.